docs/modules/operation/pages/deep-dive/alarms/alarm-example.adoc

```diff
@@ -0,0 +1,39 @@
+
+== Single alarm tracking problem states
+
+An alarm is instantiated on the first occurrence of a `service-down` problem:
+
+.`service-down` alarm created
+image::alarms/single_alarm_1.png["Alarm List displaying one alarm of minor severity"]
+
+Click the number displayed in the *Count* column to display the corresponding events and their details:
+
+.Alarm displaying service down event
+image::alarms/single_alarm_2.png["Alarm Details page displaying a service down event"]
+
+The alarm is cleared immediately when service is restored, and no alarm is created when the service returns to a normal state:
+
+.Alarm cleared
+image::alarms/single_alarm_3.png["Alarm List displaying one cleared alarm and its log message"]
+
+.Service down and service restored events
+image::alarms/single_alarm_4.png["Alarm Details page displaying one service down event and one service restored event"]
+
+If the problem occurs again, the events are reduced into the existing alarm.
```

Comment:
Not sure if "reduced" is the right word here. "rolled into"? "added to"? "integrated"? "displayed"?

Reply:
Maybe `referenced` or `linked`?

Reply:
Based on the `reduction_key`, events are grouped into a single alarm.  Reduce is the best verb in my opinion.